okhttp/src/main/java/com/squareup/okhttp/HttpUrl.java

```diff
@@ -129,8 +130,78 @@
  * TODO.
  *
  * <h3>Why another URL model?</h3>
- * TODO.
+ * Java includes both {@linkplain URL java.net.URL} and {@linkplain URI java.net.URI}. We offer a
```

Comment:
Just to confirm, intentionally not using monospace characters and linking the fully-qualified name here?